### DISCUSSION TITLE
Own house

### DISCUSSION THREAD

**Discussion by hernessteve** [NAIST-SE/AttractingContributors#990] [General]
Do you guys have your own house?
How were you able to afford it?
Isn't this just too much to take a mortgage for so many years?

**Comment by MirkosMangle** [comment]
We live in our own apartment . It is kinda small, but i hope we can afford a bigger one soon.
Don't see anything wrong about the mortagage.

**Comment by ellielo** [comment]
We are checking some loan/mortgage options at the moment with the [specialized loan servicing](https://specialized-loan-servicing.pissedconsumer.com/review.html). We are already paying a lot to rent the apartment, wouldn't be better to pay for your own house instead. At least in the second case you can leave something for your kids in the future.
